docs/application/dotnet/guides/nui/nui-components/CheckBox.md

```diff
@@ -0,0 +1,78 @@
+# CheckBox
+CheckBox is a common component and describes what action will occur when you select it.  
+A CheckBox can only contain an icon, and can be created using style.
+
+![CheckBox1](./media/CheckBox1.png)
+
+## Create with Style
+
+To create a CheckBox applying style, follow these steps:
```

Comment:
Please verify if the following suggestion is fine:

To create a checkbox using style, follow these steps:

Please verify

Reply:
fixed